evap/evaluation/templates/evap_evaluation_edit_js.html

```diff
@@ -1,32 +1,32 @@
 {% load static %}
 
 {% if editable %}
-    <script src="{% static 'js/sortable_form.js' %}"></script>
-    <script type="text/javascript">
-        rowChanged = function(row) {
-            name = $(row.find("select[id$=-contributor]")).find(":selected").text();
-            nameChanged = name && name != "---------";
-            idChanged = row.find("input[id$=-id]").val();
+    <script type="module">
+        import { makeFormSortable } from "{% static 'js/sortable-form.js' %}";
+        const rowChanged = function(row) {
+            const name = row.querySelector("select[id$=-contributor]").querySelector(":selected").textContent;
```

Comment:
Related to https://github.com/e-valuation/EvaP/pull/2199#pullrequestreview-2164156651

this fails for me, when submitting the form with
> SyntaxError: Element.querySelector: ':selected' is not a valid selector
```suggestion
            const name = row.querySelector("select[id$=-contributor]").querySelector("[selected]").textContent;
```
or just
```suggestion
            const name = row.querySelector("select[id$=-contributor] > [selected]").textContent;
```
should work instead